src/vivarium/framework/results/context.py

```diff
@@ -162,38 +166,43 @@ def _get_stratifications(
     def _format_results(
         measure: str,
         aggregates: pd.Series,
+        has_stratifications: bool,
         **additional_keys: str,
     ) -> Dict[str, float]:
         results = {}
-        # First we expand the categorical index over unobserved pairs.
-        # This ensures that the produced results are always the same length.
-        if isinstance(aggregates.index, pd.MultiIndex):
-            idx = pd.MultiIndex.from_product(
-                aggregates.index.levels, names=aggregates.index.names
-            )
+        # Simpler formatting if we don't have stratifications
+        if not has_stratifications:
+            return {measure: aggregates.squeeze()}
         else:
-            idx = aggregates.index
-        data = pd.Series(data=0, index=idx)
-        data.loc[aggregates.index] = aggregates
-
-        def _format(field, param):
-            """Format of the measure identifier tokens into FIELD_param."""
-            return f"{str(field).upper()}_{param}"
-
-        for categories, val in data.items():
-            if isinstance(categories, str):  # handle single stratification case
-                categories = [categories]
-            key = "_".join(
-                [_format("measure", measure)]
-                + [
-                    _format(field, category)
-                    for field, category in zip(data.index.names, categories)
-                ]
-                # Sorts additional_keys by the field name.
-                + [
-                    _format(field, category)
-                    for field, category in sorted(additional_keys.items())
-                ]
-            )
-            results[key] = val
-        return results
+            # First we expand the categorical index over unobserved pairs.
```

Comment:
nit: Could we do away with this `else` block and it's indentation? If the above conditional is true, we'll not enter the rest of the method.